### PR TITLE
load lib from env

### PR DESCRIPTION
**Pull request by Sasasu**
## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: src/bin/initdb/initdb.c [hunk 1]
```diff
@@ -2665,7 +2666,8 @@ usage(const char *progname)
 	printf(_("\nLess commonly used options:\n"));
 	printf(_("  -d, --debug               generate lots of debugging output\n"));
 	printf(_("  -k, --data-checksums      use data page checksums\n"));
-	printf(_("  --shared-preload-libraries load libraries, same as shared_preload_libraries in postgresql.conf\n"));
+	printf(_("  --shared-preload-libraries load libraries, same as shared_preload_libraries in postgresql.conf.\n"
```
Comment (beeender) [inline src/bin/initdb/initdb.c]: Do we have good reasons to keep the `--shared-preload-libraries`?

Comment (Sasasu) [reply src/bin/initdb/initdb.c]: show what we modified in the help message

Comment (beeender) [reply src/bin/initdb/initdb.c]: Just remove `--shared-preload-libraries`. The env should be documented on our website, and a unique way for users to set it should be good enough (and less confusion).

**Review by beeender — APPROVED** [review]

**Comment by Sasasu** [comment]
move to github.com/Sasasu/gpdb/pull/12

**Pull request closed without merging**